Dockerfile

```diff
@@ -6,12 +6,15 @@ COPY EFAuditer/EFAuditer.csproj ./EFAuditer/
 RUN dotnet restore EFAuditer/EFAuditer.csproj
 
 # restore app (in a separate layer to speed up the build)
+COPY ./NuGet.Config ./NuGet.Config
+COPY ./frontend-dotnetcore ./frontend-dotnetcore
```

Comment:
Hmm, I wonder if we should copy just the built package? Might reduce image sizes a bit?